roles/matrix-bridge-mx-puppet-twitter/tasks/init.yml

```diff
@@ -68,10 +68,3 @@
       URL endpoint to the matrix-mx-puppet-twitter container.
       You can expose the container's port using the `matrix_mx_puppet_twitter_container_http_host_bind_port` variable.
   when: "matrix_mx_puppet_twitter_enabled|bool and matrix_nginx_proxy_enabled is not defined"
-
```

Comment:
Why does this PR remove these tasks from other roles?

Reply:
They are redundant. There are two duplicate checks in each of those files.

Reply:
Oh, interesting! :laughing: